pandas_flavor/register.py

```diff
@@ -16,14 +23,39 @@ def print_column(df, col):
             print(df[col])
     """
 
+    method_signature = inspect.signature(method)
+
     def inner(*args, **kwargs):
         class AccessorMethod(object):
             def __init__(self, pandas_obj):
                 self._obj = pandas_obj
 
             @wraps(method)
             def __call__(self, *args, **kwargs):
```

Comment:
The new version of `__call__` has become more complex than before. Previously, we didn't have a docstring; while not optimal, it's still okay b/c of the relative simplicity of the original implementation. With the additional code, though, I think we should add a docstring. 

Here is something that I used Tabnine to generate as a starter:

```python
                """
                Call the method with the given arguments and return the result.

                This method is called when the user calls the method on the
                accessor object. It calls the underlying method with the
                arguments and keyword arguments passed to it.

                The method_call_context_factory is used
                to record the methods that were called.

                Parameters
                ----------
                *args : list
                    The arguments to pass to the method.
                **kwargs : dict
                    The keyword arguments to pass to the method.

                Returns
                -------
                object :
                    The result of calling the method.
                """
```

Would you be open to modifying it to describe the behaviour of `method_call_ctx_factory accurately`? (I will admit that I lost context (pun unintended) on pyjviz because of the myriad of activities happening at home, so recording it in-line here can be helpful.)

Reply:
will do, thanks for the feedback